src/Mod/PartDesign/Gui/TaskPipeParameters.cpp

```diff
@@ -87,12 +87,12 @@ TaskPipeParameters::TaskPipeParameters(ViewProviderPipe *PipeView, bool /*newObj
             this, SLOT(onProfileButton(bool)));
     connect(ui->comboBoxTransition, SIGNAL(currentIndexChanged(int)),
             this, SLOT(onTransitionChanged(int)));
-    connect(ui->buttonRefAdd, SIGNAL(toggled(bool)),
-            this, SLOT(onButtonRefAdd(bool)));
-    connect(ui->buttonRefRemove, SIGNAL(toggled(bool)),
-            this, SLOT(onButtonRefRemove(bool)));
-    connect(ui->buttonSpineBase, SIGNAL(toggled(bool)),
-            this, SLOT(onBaseButton(bool)));
+    // connect(ui->buttonRefAdd, SIGNAL(toggled(bool)),

```

Comment:
why do you keep them? They are not needed anymore and a comment why there are no connections should be sufficient.